integration-tests/testsetups/ocr.go

```diff
@@ -270,107 +273,151 @@ func (o *OCRSoakTest) Environment() *environment.Environment {
 	return o.testEnvironment
 }
 
+// Setup initializes the OCR Soak Test by setting up clients, funding nodes, and deploying OCR contracts.
 func (o *OCRSoakTest) Setup(ocrTestConfig tt.OcrTestConfig) {
+	o.initializeClients()
+	o.deployLinkTokenContract(ocrTestConfig)
+	o.fundChainlinkNodes()
+
+	o.startingValue = 5
+
+	var forwarders []common.Address
+	if o.OperatorForwarderFlow {
+		_, forwarders = o.deployForwarderContracts()
+	}
+
+	o.setupOCRContracts(ocrTestConfig, forwarders)
+	o.log.Info().Msg("OCR Soak Test Setup Complete")
+}
+
+// initializeClients sets up the Seth client, Chainlink nodes, and mock server.
+func (o *OCRSoakTest) initializeClients() {
 	sethClient, err := seth_utils.GetChainClient(o.Config, o.rpcNetwork)
 	require.NoError(o.t, err, "Error creating seth client")
 	o.seth = sethClient
 
 	nodes, err := nodeclient.ConnectChainlinkNodes(o.testEnvironment)
 	require.NoError(o.t, err, "Connecting to chainlink nodes shouldn't fail")
 	o.bootstrapNode, o.workerNodes = nodes[0], nodes[1:]
+
 	o.mockServer = ctf_client.ConnectMockServer(o.testEnvironment)
 	require.NoError(o.t, err, "Creating mockserver clients shouldn't fail")
+}
 
-	linkContract, err := actions.LinkTokenContract(o.log, sethClient, ocrTestConfig.GetActiveOCRConfig())
+func (o *OCRSoakTest) deployLinkTokenContract(ocrTestConfig tt.OcrTestConfig) {
+	linkContract, err := actions.LinkTokenContract(o.log, o.seth, ocrTestConfig.GetActiveOCRConfig())
 	require.NoError(o.t, err, "Error loading/deploying link token contract")
+	o.linkContract = linkContract
+}
 
-	// Fund Chainlink nodes, excluding the bootstrap node
+// fundChainlinkNodes funds the Chainlink worker nodes.
+func (o *OCRSoakTest) fundChainlinkNodes() {
 	o.log.Info().Float64("ETH amount per node", *o.Config.Common.ChainlinkNodeFunding).Msg("Funding Chainlink nodes")
-	err = actions.FundChainlinkNodesFromRootAddress(o.log, sethClient, contracts.ChainlinkK8sClientToChainlinkNodeWithKeysAndAddress(o.workerNodes), big.NewFloat(*o.Config.Common.ChainlinkNodeFunding))
+	err := actions.FundChainlinkNodesFromRootAddress(o.log, o.seth, contracts.ChainlinkK8sClientToChainlinkNodeWithKeysAndAddress(o.workerNodes), big.NewFloat(*o.Config.Common.ChainlinkNodeFunding))
```

Comment:
I'd discourage naming a field/variable in the same way as the package is named (as you might get weird compilation errors). maybe just rename it to `ethereumClient`, it shouldn't really matter whether it's Seth or not and it will be obvious from the rest of the code.

Reply:
Done. I have updated the OCRSoakTest struct field to sethClient from seth.